NewHorizons/Builder/Atmosphere/FogBuilder.cs

```diff
@@ -19,7 +19,6 @@ public static void Make(GameObject planetGO, Sector sector, AtmosphereModule atm
             // Going to copy from dark bramble
             var dbFog = SearchUtilities.Find("DarkBramble_Body/Atmosphere_DB/FogLOD");
             var dbPlanetaryFogController = SearchUtilities.Find("DarkBramble_Body/Atmosphere_DB/FogSphere_DB").GetComponent<PlanetaryFogController>();
-            var brambleLODFog = SearchUtilities.Find("DarkBramble_Body/Sector_DB/Proxy_DB/LOD_DB_VolumeticFog");
```

Comment:
Yes it was intentional. It was removed because the LOD fog does this sharp pop.
Plus, it isn't used on any of vanilla planets (except dark bramble).

https://user-images.githubusercontent.com/34462599/178417674-64c07a4b-fac6-4a07-9a31-4ef135c0ac9c.mp4